pyuvdata/uvdata/corr_fits_src/corr_fits.pyx

```diff
@@ -95,30 +96,35 @@ cpdef tuple generate_map(
             )
             map_inds[bls_ind * 4 + pol_ind] = data_index
 
-  return map_inds, conj
+  return
 
+@cython.boundscheck(False)
+@cython.wraparound(False)
 cpdef numpy.ndarray[ndim=1, dtype=numpy.float64_t] get_cable_len_diffs(
   int Nblts,
-  numpy.ndarray[dtype=numpy.int_t, ndim=1] ant1_array,
-  numpy.ndarray[dtype=numpy.int_t, ndim=1] ant2_array,
+  numpy.int_t[:] ant1_array,
+  numpy.int_t[:] ant2_array,
   numpy.ndarray cable_lens,
 ):
-  cdef int i
-  cdef numpy.ndarray[ndim=1, dtype=numpy.float64_t] cable_array = np.zeros(len(cable_lens), dtype=np.float_)
-  cdef numpy.ndarray[ndim=1, dtype=numpy.float64_t] cable_diffs = np.zeros(Nblts, dtype=np.float_)
-
+  cdef Py_ssize_t i
+  cdef int n_cables = cable_lens.shape[0]
+  cdef numpy.float64_t[::1] cable_array = np.zeros(n_cables, dtype=np.float_)
```

Comment:
I know that `np.float_` is the same thing as `np.float64`, but I have a slight preference for being explicit in the number of bits (though I understand I'm a bit of an old-timer in that regard). Also a big fan of using memory views 👍

Reply:
I think I remember the use of `np.float_` having a difference precision on windows too. Thanks for pointing this out, I'll try to change any instances of it.